src/wasm/wasm-s-parser.cpp

```diff
@@ -1614,7 +1619,7 @@ void SExpressionWasmBuilder::parseImport(Element& s) {
       }
       if (inner.size() > j+1) {
         Element& result = *inner[j+1];
-        assert(result[0]->str() == RESULT);
+        if (result[0]->str() != RESULT)  throw ParseException("expected result");
```

Comment:
Nit: Double-spacing before `throw`

As an aside: yay for changing asserts to errors with messages!